### PR TITLE
trigger api documentation

### PR DESCRIPTION
**Pull request by helenaford**

### CODE REVIEW

File: docs/react-native/docs/triggers.md [hunk 1]
```diff
@@ -1,8 +1,100 @@
...
-title: Trigger
-description: Trigger notifications to appear under specific commands such as scheduling.
+title: Triggers
+description: Understand how to use triggers to create notifications that fire under specific conditions.
 next: /react-native/docs/integrations/fcm
 previous: /react-native/docs/events
 ---
 
-This documentation is a work in progress.
+Triggers can be used to display notifications in-advance when a specific condition is met such as time or location.
```
Comment (Ehesp) [inline docs/react-native/docs/triggers.md]: `or location.` Is it worth removing this until its supported?

File: docs/react-native/docs/triggers.md [hunk 2]
```diff
@@ -1,8 +1,100 @@
...
+> Currently only supported on Android, but iOS will be coming soon.
+
+# Creating a trigger notification
+
+```js
+function Screen() {
+  async function onCreateTriggerNotification() {
+    // Create a time-based trigger
+    const trigger: TimeTrigger = {
+      type: TriggerType.TIME,
```
Comment (Ehesp) [inline docs/react-native/docs/triggers.md:24]: Should we show the import above?

Comment (helenaford) [reply docs/react-native/docs/triggers.md:24]: ah yeah, good shout

File: docs/react-native/docs/triggers.md (hunk 1 above)
Comment (helenaford) [reply docs/react-native/docs/triggers.md]: I mention it below it's not supported but yeah maybe if it's confusing

**Pull request merged**